db/migrate/20190130140611_associate_filters_with_accounts.rb

```diff
@@ -0,0 +1,7 @@
+class AssociateFiltersWithAccounts < ActiveRecord::Migration[5.2]
+  def change
+    change_table :filters do |t|
+      t.references :account, :null => false
```

Comment:
This migration creates a `bigint` column in DB. Which is not what we need, since `accounts.id` is UUID